src/utils/index.py

```diff
@@ -56,7 +56,9 @@ def index_files(
     ursa_url, types, tags, batch, compact_threshold = proc_params
     ursa = UrsaDb(ursa_url)
 
-    current_datasets = len(ursa.topology()["result"]["datasets"])
+    current_datasets = len(
+        ursa.execute_command("topology;")["result"]["datasets"]
```

Comment:
`UrsaDb.topology`, `UrsaDb.status` and `UrsaDb.execute_command` have almost the same code (the only difference is turned off timeout in the last one). Maybe should be refactored in the future?

Reply:
:weltschmerz2: yeah, good idea